src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Data/ReviewSqlQueriesForSecurityVulnerabilities.cs

```diff
@@ -94,9 +94,11 @@ public override void Initialize(AnalysisContext context)
                             return;
                         }
 
+                        M();
                         AnalyzeMethodCall(operationContext, invocation.TargetMethod, symbol, invocation.Arguments, invocation.Syntax, isInDbCommandConstructor, isInDataAdapterConstructor, iDbCommandType, iDataAdapterType);
                     }, OperationKind.Invocation);
 
+                    static bool M() => true;
```

Comment:
😕 The fix all changed this?

Reply:
No, I was testing something before working on this branch and forgot to revert it :(

Reply:
Reverted in https://github.com/dotnet/roslyn-analyzers/pull/4086/commits/7b7c30ae5efefa32118df1096a1969f77e4780a0